apollo-router/src/plugins/traffic_shaping/mod.rs

```diff
@@ -494,7 +494,7 @@ mod test {
             .oneshot(SubgraphRequest::fake_builder().build())
             .await
             .unwrap();
-        tokio::time::sleep(Duration::from_millis(300)).await;
+        tokio::time::sleep(Duration::from_millis(310)).await;
```

Comment:
What happened here?

Reply:
As per commit comment: https://github.com/apollographql/router/pull/1547/commits/7aad3bfe9f79eaefda9afcbe3c0775df81467726

Reply:
I've reverted this in my latest merge with main since it doesn't seem to have improved things.